data_and_models/pipelines/sentence_embedding/dvc.yaml

```diff
@@ -26,7 +27,8 @@ stages:
     - ../../metrics/sentence_embedding/biobert_nli_sts.csv
     - ../../metrics/sentence_embedding/biobert_nli_sts.png
```

Comment:
`.png` outputs are pretty quick to recreate, and at the same time each of these output images takes ~0.2 MB (on the other hand, `.csv` are just ~4 KB each).

So what about declaring the `.png` outputs as `cache: false` as well?

Reply:
Would this mean we would track these PNG-files in our GitHub repository instead of the DVC remote?